doc/content/reference/api/end_device.md

```diff
@@ -287,6 +287,8 @@ See the [EndDevice message](#message:EndDevice) and its sub-messages for additio
 
 {{< proto/message message="Location" >}}
 
+{{< proto/message message="LoRaAllianceProfileIdentifiers" >}}
```

Comment:
These are changes from `v3.24`. Are you sure you run the regeneration from the `v3.23.2` tag ?